CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 Changelog
+2.0
+- Renamed this project 'OpenCompliance' to encompass CCPA support
```

Comment:
I think `OpenDPCompliance` or `OpenDataProtectionCompliance` would be a better name since this project only deals with data protection compliance.

Reply:
The scoping is tough and the words are often vague. What do you think about:
- OpenDSR
- OpenDataPrivacy
- OpenPrivacyCompliance

Reply:
I like all three. Maybe OpenDSR is the shortest. What do you think?

Reply:
`OpenDSR` makes total sense for our current scope, but `OpenDataPrivacy` give us a little wiggle room to do other adjacent things without renaming, eg if we wanted to add a format for consent/opt-out.  `OpenDSR` is probably the best and we can grow later if we need

Reply:
Going with OpenDSR, updated PR incoming

Reply:
Sry for the late response. You mean that `OpenDataPrivacy` would be better, if the framework would be extended to other privacy related stuff right?

Reply:
Yes, that is what I meant. Upon reflection it might make sense to scope additional frameworks separately by topic instead of continually expanding this one.  I could see a similar project for OpenConsent to standardize consent structures.